x-pack/plugins/apm/public/components/fleet_integration/apm_agents/agent_instructions_accordion.tsx

```diff
@@ -235,7 +235,12 @@ export function AgentInstructionsAccordion({
                   >
                     <EuiFlexItem grow={false}>Auto-Attachment</EuiFlexItem>
                     <EuiFlexItem grow={false}>
-                      <EuiBetaBadge label="BETA" />
+                      <EuiBetaBadge
+                        label={i18n.translate(
+                          'xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.betaBadge.label',
+                          { defaultMessage: 'BETA' }
+                        )}
+                      />
```

Comment:
Did we loose the `tooltipContent` I suggested previously here?

```
tooltipContent={i18n.translate('xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.betaBadge.tooltipContent',
              {
                defaultMessage:
                  'Auto-attachment for Java is not GA. Please help us by reporting any bugs.',
              }
            )}
```

Reply:
It's back, my bad 
<img width="390" alt="Screen Shot 2021-12-07 at 11 13 59 AM" src="https://user-images.githubusercontent.com/55978943/145065530-a9cf89b6-d210-437f-a349-a296dde651cf.png">